api-metastore/src/main/java/org/zalando/nakadi/controller/SchemaController.java

```diff
@@ -52,9 +63,25 @@ public ResponseEntity<?> create(@PathVariable("name") final String name,
             throw new ValidationException(errors);
         }
 
-        schemaService.addSchema(name, schema);
-
-        return status(HttpStatus.OK).build();
+        final var originalEventType = eventTypeService.fetchFromRepository(name);
+        schemaService.addSchema(originalEventType, schema)
```

Comment:
The service could provide both original and updated event type objects in the return value.  This will remove the need for the new method.

At the same time...  I think I made a very similar change in my PR (but for a different reason): https://github.com/zalando/nakadi/pull/1422/files#diff-5a8af8d6f60a128e59e74ed1304d8f16cdcce045cf29b71b84e35190b1530bd2R65